### DISCUSSION TITLE
How to name route file in folder other than index.tsx

### DISCUSSION THREAD

**Discussion by juniorforlife** [remix-run/remix#6311] [Q&A]
I'm using flat route and end up with a lot of index files. Do we have a way to name these files in a more declarative way?
For example Solid Start uses parenthesis (home).tsx for naming index routes

**Comment by kiliman** [comment]
Not in `v2` routes, but you can with `remix-flat-routes`.

Any route file that starts with `_` is treated as the route module, similar to Solid Starts parenthesis.

https://github.com/kiliman/remix-flat-routes#extended-route-filenames

**Reply by juniorforlife** [reply]
thanks! Please add this feature in Remix v2 as well 🙏 . It's just chaos to find a route when everything is index

**Reply by kiliman** [reply]
In VS Code, remember when you do Quick Open (Ctrl/Cmd+P), you can type any part of the full path so that you can type in the folder name and find the file that way.
